### PR TITLE
Add / update definition, summary metrics and golden metrics for several AWS entity types: EFS, Elasticache, Elasticsearch, ELB, Kinesis

### PR DESCRIPTION
**Pull request by ramonguiu**
### Relevant information
Add / update definition, summary metrics and golden metrics for several AWS entity types: EFS, Elasticache, Elasticsearch, ELB, Kinesis.

### Checklist

* [x] I've read the guidelines and understand the acceptance criteria.
* [x] The value of the attribute marked as `identifier` will be unique and valid. 
* [x] I've confirmed that my entity type wasn't already defined. If it is I'm providing an
 explanation above.

### CODE REVIEW

**Comment by ramonguiu** [comment]
I'm not sure where the error is after checking the output of the tests. If you could provide guidance happy to take a look.

File: definitions/infra-awselasticsearchnode/definition.yml [hunk 1]
```diff
@@ -1,5 +1,10 @@
 domain: INFRA
 type: AWSELASTICSEARCHNODE
+cgoldenTags:
```
Comment (srvaroa) [inline definitions/infra-awselasticsearchnode/definition.yml:3]:
```suggestion
goldenTags:
```

This should fix it

File: definitions/infra-awselasticsearchnode/summary_metrics.yml [hunk 2]
```diff
@@ -6,14 +6,21 @@ providerAccountName:
 readThroughput:
   query:
     eventId: entityGuid
-    select: (average(`provider.IndexingRate.Average`)) / 60
+    select: (average(`provider.IndexingRate.Average`))
     from: DatastoreSample
-  unit: REQUESTS_PER_SECOND
+  unit: REQUESTS_PER_MINUTE
```
Comment (naxhh) [inline definitions/infra-awselasticsearchnode/summary_metrics.yml:11]: This is not a valid unit. Please check [our docs](https://github.com/newrelic-experimental/entity-synthesis-definitions/blob/main/docs/summary_metrics.md#metric-unit)

File: definitions/infra-awselb/summary_metrics.yml [hunk 3]
```diff
@@ -3,24 +3,24 @@ providerAccountName:
...
+throughput:
   query:
     eventId: entityGuid
-    select: sum(`provider.requestCount.Sum`)
+    select: rate(sum(`provider.requestCount.Sum`), 1 minute)
     from: LoadBalancerSample
-  unit: COUNT
-  title: Requests
-healthyHosts:
+  unit: REQUESTS_PER_MINUTE
```
Comment (naxhh) [inline definitions/infra-awselb/summary_metrics.yml:11]: same as before

**Comment by jfjoly** [comment]
As discussed with @naxhh , we will add support for `REQUESTS_PER_MINUTE`

**Comment by jfjoly** [comment]
To clarify, the conversion will be done in the UI.

**Comment by naxhh** [comment]
So after @ramonguiu can move back to `REQUESTS_PER_SECOND` and sign the CLA we will be ready to merge!

**Review by naxhh — APPROVED** [review]

**Review by srvaroa — APPROVED** [review]

**Pull request merged**